requirements/constraints.txt

```diff
@@ -12,15 +12,11 @@
 # This file contains all common constraints for edx-repos
 -c common_constraints.txt
 
-# celert>5.0.0 hasn't been tested yet, so the constraint will be removed after testing latest version
-celery<5.0.0
+celery>=5.2.1,<5.3.0
```

Comment:
Please add a comment to explain the 5.3.0 constraint. Thanks.

Reply:
1. Thanks for this comment. It is useful to know that it is not being constrained for any known issues.
2. Is celery known to introduce breaking changes on minor versions? That's not nice. I don't know about celery, but typically we either leave it open or might put in something like `<6.0.0` with this comment, because that's when you'd expect breaking changes.

That being said, if celery upgrades have more risk for some reason, we could leave this as-is.

Reply:
I’m not sure about what I wrote. Celery is not used to break things in patch/minor versions, so the constraint should be `celery>=5.2.0,<6.0.0`. I just adjusted it.

Reply:
Given the https://github.com/advisories/GHSA-q4xr-rc97-m4xx vulnerability in Celery < 5.2.2, could you please change the lower limit to `celery>=5.2.2` ([Celery changelog v5.2.2](https://github.com/celery/celery/blob/master/Changelog.rst#522))?